### PR TITLE
Fixes #10094: Improve new Rules interface

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://www.rudder-project.org/redmine/issues/10094

### CODE REVIEW

**Comment by RaphaelGauthier** [comment]
PR rebased

File: rudder-web/src/main/webapp/javascript/AdminLTE/AdminLTE.min.js [hunk 1]
```diff
@@ -10,4 +10,4 @@
...
...xtend({trigger:".refresh-btn",source:"",onLoadStart:function(a){return a},onLoadDone:function(a){return a}},b),f=a('<div class="overlay"><div class="fa fa-refresh fa-spin"></div></div>');return this.each(function(){if(""===e.source)return void(window.console&&window.console.log("Please specify a source first - boxRefresh()"));var b=a(this),f=b.find(e.trigger).first();f.on("click",function(a){a.preventDefault(),c(b),b.find(".box-body").load(e.source,function(){d(b)})})})}}(jQuery),function(a){"use strict";a.fn.activateBox=function(){a.AdminLTE.boxWidget.activate(this)}}(jQuery),function(a){"use strict";a.fn.todolist=function(b){var c=a.extend({onCheck:function(a){return a},onUncheck:function(a){return a}},b);return this.each(function(){"undefined"!=typeof a.fn.iCheck?(a("input",this).on("ifChecked",function(){var b=a(this).parents("li").first();b.toggleClass("done"),c.onCheck.call(b)}),a("input",this).on("ifUnchecked",function(){var b=a(this).parents("li").first();b.toggleClass("done"),c.onUncheck.call(b)})):a("input",this).on("change",function(){var b=a(this).parents("li").first();b.toggleClass("done"),a("input",b).is(":checked")?c.onCheck.call(b):c.onUncheck.call(b)})})}}(jQuery);
```
Comment (VinceMacBuche) [inline rudder-web/src/main/webapp/javascript/AdminLTE/AdminLTE.min.js]: Have you upgraded AdminLTE.min.js ?

Comment (VinceMacBuche) [inline rudder-web/src/main/webapp/javascript/AdminLTE/AdminLTE.min.js]: If you changed it' version you should changed it in comment

If you made a change inside you should comment it and put the diff here or have the modified source file (non minified file)

Comment (VinceMacBuche) [inline rudder-web/src/main/webapp/javascript/AdminLTE/AdminLTE.min.js]: Ok you changed some icon, can you put that in comment ?

Comment (RaphaelGauthier) [inline rudder-web/src/main/webapp/javascript/AdminLTE/AdminLTE.min.js]: Done! I added the modified source file too.

**Comment by RaphaelGauthier** [comment]
Commit modified

File: rudder-web/src/main/webapp/javascript/AdminLTE/AdminLTE.min.js (hunk 1 above)
Comment (VinceMacBuche) [inline rudder-web/src/main/webapp/javascript/AdminLTE/AdminLTE.min.js]: great !

**Pull request merged**